cmd/token.go

```diff
@@ -50,17 +50,32 @@ var (
 	}
 
 	tokenCreateCmd = &cobra.Command{
-		Use:   "create",
+		Use:   "create --read|-r | --write|-w | --modify|-m | --stage|-s <pelican-url>",
 		Short: "Create a token",
 		RunE:  createToken,
 		Args:  cobra.ExactArgs(1),
 		Example: "To create a read/write token for /some/namespace/path in OSDF: " +
 			"pelican token create --read --write pelican://osg-htc.org/some/namespace/path",
 	}
+
+	tokenFetchCmd = &cobra.Command{
+		Use:   "fetch --write|-w | --modify|-m <pelican-url>",
+		Short: "Fetch a token",
+		RunE:  fetchToken,
+		Args:  cobra.ExactArgs(1),
+		Example: "To fetch a token for /some/namespace/path in OSDF: " +
+			"pelican token fetch pelican://osg-htc.org/some/namespace/path",
+		Hidden: true,
+	}
 )
 
 func init() {
 	tokenCmd.AddCommand(tokenCreateCmd)
+	tokenCmd.AddCommand(tokenFetchCmd)
+
+	// Token fetch capabilities
+	tokenFetchCmd.Flags().BoolP("write", "w", false, "Fetch a token with the ability to write to or modify the specified resource. ")
```

Comment:
Let's also have a `--read` flag so that token scopes are explicitly requested (the user has to know what they're asking for).

In fact, if these are the same exact flags used with the token create command, create a set of flags and apply them to both commands so we don't have two sets to manage.